import_3dm/converters/render_mesh.py

```diff
@@ -73,10 +75,26 @@ def import_render_mesh(context, ob, name, scale, options):
 
         fidx = fidx + len(m.Vertices)
         vertices.extend([(m.Vertices[v].X * scale, m.Vertices[v].Y * scale, m.Vertices[v].Z * scale) for v in range(len(m.Vertices))])
-    
+        coords.extend([(m.TextureCoordinates[v].X, m.TextureCoordinates[v].Y) for v in range(len(m.TextureCoordinates))])
+   
     mesh = context.blend_data.meshes.new(name=name)
     mesh.from_pydata(vertices, [], faces)
 
+
+    if(mesh.polygons and coords):
```

Comment:
No need to enclose in parentheses

Reply:
haha, wrote a lot of c# the last weeks, must have been out of habit ;)